CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 This document outlines major changes between releases.
 
+## 0.18.0 (16 Dec 2021)
+
+### Added
+- Support for MultipartUpload (#186, #187) 
+- CORS support (#217)
+- Authmate supports setting of tokens' lifetime in more convenient format (duration) (#258)
```

Comment:
Maybe something like this would be better:
```suggestion
- Authmate supports setting of tokens lifetime in more convenient format (duration) (#258)
```

Reply:
Why? We can put multiple tokens (bearer and session) and tokens' lifetime means 'lifetime of tokens'.

Reply:
If I understand correctly `tokens lifetime` means the same as `lifetime of tokens`

Reply:
I don't thinks an apostrophe is necessary here. It's kinda hard to explain, but I don't feel like "tokens" own "lifetime". In the singular form, I would say `a token lifetime` without `'s`-identified possession. Thus, we don't need it in the plural form; so I would prefer: `Authmate supports setting of tokens lifetime in **a** more convenient format`. 
Though, if you doubt, you can always choose the `lifetime of tokens` form.
p.s. I also googled some phrases like "password lifetime" or "account lifetime" and saw no "'s".